src/app/api/auth/lib/nextauth-options.ts

```diff
@@ -22,6 +23,67 @@ export const verifySession = async (token: string | undefined) => {
   }
 }
 
+/**
+ * Refresh access token
+ * @param clientId Client ID
+ * @param clientSecret Client Secret
+ * @param refreshToken Refresh token
+ * @returns object — New access token and refresh token
+ */
+export const refreshAccessToken = async (
+  token: JWT,
+  clientId: string,
+  clientSecret: string,
+  refreshToken: string,
+) => {
+  try {
+    authLogger.debug('Refreshing access token')
```

Comment:
Could we include clientId for more specific tracking? I imagine this would highly benefit security inclined users.
```suggestion
    authLogger.debug('Refreshing access token', clientId)
```

Reply:
Is the clientId a secret that we wouldn't want stored in the logs or is it safe to show?